File: Arrays/Operations/update.java

package Operations;

public class update {

    public static void main(String[] args) {

        int[] arr = {10, 20, 30, 40, 50};

        int index = 2;
        int newValue = 35;

        arr[index] = newValue;

        System.out.println("Array after update:");

        for (int i = 0; i < arr.length; i++) {
            System.out.print(arr[i] + " ");
        }
    }
}

File: Arrays/Operations/update.c
#include <stdio.h>

int main() {

    int arr[] = {10, 20, 30, 40, 50};
    int n = 5;

    int index = 2;
    int newValue = 35;

    arr[index] = newValue;

    printf("Array after update:\n");

    for (int i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }

    return 0;
}